languages/julia/exercises/concept/unicode-identifiers/.meta/design.md

```diff
@@ -24,6 +24,7 @@
 
 This exercise's prerequisites Concepts are:
 
+- Number types (TODO: Depends on what the concepts will be called)
```

Comment:
It doesn't matter too much what type they use internally to represent the clock state but it will likely require some kind of number.